app.js

```diff
@@ -8,11 +8,17 @@ const app = express()
 
 const formatsLogger = app.get('env') === 'development' ? 'dev' : 'short'
 
+
+
```

Comment:
Staraj się unikać pustych linii. Jedyna taka odległość Jaka może być to jest jedna linia i to dobrze wyglądam co więcej no to już nie bardzo.